pkg/controller/kogitoinfra/infinispan.go

```diff
@@ -122,35 +143,114 @@ func getInfinispanAppProps(cli *client.Client, name string, namespace string) (m
 	}
 	appProps[propertiesInfinispanSpring[appPropInfinispanSaslMechanism]] = saslPlain
 	appProps[propertiesInfinispanQuarkus[appPropInfinispanSaslMechanism]] = saslPlain
+
+	if len(i.instance.Status.Volume) > 0 {
+		appProps[propertiesInfinispanSpring[appPropInfinispanTrustStoreType]] = pkcs12CertType
+		appProps[propertiesInfinispanQuarkus[appPropInfinispanTrustStoreType]] = pkcs12CertType
+		appProps[propertiesInfinispanSpring[appPropInfinispanTrustStore]] = certMountPath + "/" + truststoreSecretKey
+		appProps[propertiesInfinispanQuarkus[appPropInfinispanTrustStore]] = certMountPath + "/" + truststoreSecretKey
```

Comment:
Wouldn't it be better to retrieve truststore path by mount path?
```suggestion
		appProps[propertiesInfinispanQuarkus[appPropInfinispanTrustStore]] = i.instance.Status.Volume[0].Mount.MountPath
```

Reply:
I will add this info in a shared constant, acessing the array directly is not reliable.